client/src/components/title/title.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+
+class title extends Component {
+	constructor(props) {
+		super(props);
+	}
```

Comment:
To make the project consistent, react functional components. 
```suggestion
function Title(props) {
....
        ----<sniplet>----
....
        return (
        <>DOM<>
        )
}
```